recipe/meta.yaml

```diff
@@ -197,15 +191,15 @@ outputs:
         - binutils_impl_{{ target_platform }}  # [linux]
 {% endif %}
       host:
-        - bzip2  # [not win]
+        - bzip2
         - sqlite
         - xz  # [not win]
         - zlib  # [not win]
         - openssl
         - readline  # [not win]
         - tk  # [not win]
         - ncurses  # [unix]
-        - libffi   # [not (win or (osx and arm64))]
+        - libffi   # [not (osx and arm64)]
```

Comment:
With libffi patch on Anaconda for arm64, it should be possible to use even for arm64 libffi package, instead of system one.  Major fix there is that the ffi_find_closure_for_code_np adjustment (Apple renamed the ffi_find_closure_for_code function)

Reply:
Thanks @katietz for the info. Can you send a link to the patch?

Reply:
Sure, it is part of this 3.3 patch for arm64.  We work right now on porting patch to 3.4 too. https://github.com/AnacondaRecipes/libffi-feedstock/blob/master/recipe/0003-libffi-3.3-arm64.patch
I used different sourceball and modified directly configuration stuff on libffi.  Sadly the apple-distribution python script has no support for arm64 for now ... therefore a reconfigure of autotool stuff would break it

Reply:
see generate-darwin-source-and-headers.py file in libffi's root directory in sourceball.  It is neither adjusted in 3.3 for arm64, nor for osx64.  But it is required to setup the header-dependencies proper.  So this patch actually did this in a manual way ...

Reply:
Thanks. I've added a much smaller patch to 3.4. See the first two patches at https://github.com/conda-forge/libffi-feedstock/tree/master/recipe